vlog4j-core/src/main/java/org/semanticweb/vlog4j/core/reasoner/QueryAnswersSize.java

```diff
@@ -0,0 +1,64 @@
+package org.semanticweb.vlog4j.core.reasoner;
+
+/*-
+ * #%L
+ * VLog4j Core Components
+ * %%
+ * Copyright (C) 2018 - 2019 VLog4j Developers
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+/**
+ * 
+ * @author Larry González
+ *
+ */
+public class QueryAnswersSize {
+
+	final Correctness correctness;
+	final long size;
+
+	public QueryAnswersSize(Correctness correctness, int size) {
```

Comment:
this constructor is not necessary, as `long` variables can be assigned `int` values.